File: CS/ColoredProgressBar/Form1.cs
using System;
using System.Data;
using System.Windows.Forms;

namespace ColoredProgressBar {
    public partial class Form1 : Form {
        DataTable dt = new DataTable();
        public Form1() {
            InitializeComponent();
            dt.Columns.Add("Column");
            for (int i = 0; i <= 100; i += 10)
                dt.Rows.Add(i);
        }
        private void Form1_Load(object sender, EventArgs e) {
            gridControl1.DataSource = dt;
            CustomPaintedProgressBarHelper customPaintedProgressBarHelper = new CustomPaintedProgressBarHelper(col4);
            DRProgressBarHelper drHelper = new DRProgressBarHelper(col5);
            SRProgressBarHelper srHelper = new SRProgressBarHelper(col6);
        }
    }
}

File: VB/ColoredProgressBar/Form1.vb
Imports System
Imports System.Data
Imports System.Windows.Forms

Namespace ColoredProgressBar

    Public Partial Class Form1
        Inherits Form

        Private dt As DataTable = New DataTable()

        Public Sub New()
            InitializeComponent()
            dt.Columns.Add("Column")
            For i As Integer = 0 To 100 Step 10
                dt.Rows.Add(i)
            Next
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs)
            gridControl1.DataSource = dt
            Dim customPaintedProgressBarHelper As CustomPaintedProgressBarHelper = New CustomPaintedProgressBarHelper(col4)
            Dim drHelper As DRProgressBarHelper = New DRProgressBarHelper(col5)
        End Sub
    End Class
End Namespace
